### ISSUE TITLE
NVDA not reading alt image values despite tabindex=0 on image

### ISSUE THREAD

**Issue by SomethingNew71** [nvaccess/nvda#7589]
### Steps to reproduce:
> Example:
>
> 1. Navigate to http://www.ally.com/ventures
> 2. Tab to Relationship Section
> 3. Use Tab or hit enter to open(flip) an image card.
> 4. The focus is being placed on image however it is not being announced.

### Expected behavior:
> Image alt text should be announced when the focus is placed on image element.

### Actual behavior:
> Image alt text is not being read despite image having a `tabindex="0"`.

### System configuration:
NVDA version:
> 2016.4

NVDA Installed or portable:
> Installed

Other information:
> Firefox Web Browser

Windows version:
> Windows 10 

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
> Yes

Have you tried any other versions of NVDA?
> None

**Comment by ehollig** [comment]
Could you please try this with the latest version of NVDA 2017.3? Performing the above steps in Firefox works correctly using the latest Master version of NVDA

**Comment by SomethingNew71** [comment]
Confirmed I just updated to 2017.3 and FF 55.0.3 its still displaying the same behavior.

**Comment by ehollig** [comment]
@SomethingNew71 when I click on the Learn more about Better button, NVDA properly places focus on, and announces the alt text of the better graphic. What exactly is the issue here?

**Comment by Adriani90** [comment]
When you are in browse mode and you move to one of the images and press enter, NVDA does not report anything. It should report the grafic together with its alt text and description. In focus mode, on the grafic and its alt text are reported but not the description.

Tested with NVDA 2019.1.1 on Firefox 66.0.3 and Chrome 74, the same behavior is visible.

cc: @jcsteh

**Comment by Malvoz** [comment]
> 4. The focus is being placed on image however it is not being announced.

This is no longer true. Focus is delegated to a heading, which is announced. Tested in 2022.1.

I suggest closing this. I'm trying to understand if my issue has been reported before, but there are too many similar issues. Perhaps I should open a new one anyway.

**Comment by seanbudd** [comment]
Closing as fixed
